pkg/apis/tensorflow/v1alpha1/types.go

```diff
@@ -58,10 +58,23 @@ type TFJobSpec struct {
 	// TerminationPolicy specifies the condition that the tfjob should be considered finished.
 	TerminationPolicy *TerminationPolicySpec `json:"terminationPolicy,omitempty"`
 
+	// CleanPodPolicy specifies the operation when tfjob is finished or failed.
+	CleanPodPolicy CleanPodPolicyType `json:"cleanPodPolicy,omitempty"`
+
 	// SchedulerName specifies the name of scheduler which should handle the TFJob
 	SchedulerName string `json:"schedulerName,omitempty"`
 }
 
+// CleanPodPolicyType defines all kinds of types of cleanPodPolicy.
+type CleanPodPolicyType string
+
+const (
+	CleanUndefined CleanPodPolicyType = ""
+	CleanAll       CleanPodPolicyType = "CleanAll"
+	CleanRunning   CleanPodPolicyType = "CleanRunning"
+	CleanNone      CleanPodPolicyType = "CleanNone"
+)
```

Comment:
I think we should follow the convention in the k8s community:

```go
// CleanPodPolicy describes how to deal with pods when the TFJob is finished.
type CleanPodPolicy string

const (
	CleanPodPolicyUndefined CleanPodPolicy = ""
	CleanPodPolicyAll       CleanPodPolicy = "All"
	CleanPodPolicyRunning   CleanPodPolicy = "Running"
	CleanPodPolicyNone      CleanPodPolicy = "None"
)
```

Thus maybe the code above is more appropriate, IMO